src/psyclone/psyir/tools/definition_use_chains.py

```diff
@@ -449,6 +458,11 @@ def _compute_forward_uses(self, basic_block_list):
                     if defs_out is not None:
                         self._defsout.append(defs_out)
                     return
+                # If its parent is an inquiry function then its neither
+                # a read nor write.
+                if (isinstance(reference.parent, IntrinsicCall) and
+                        reference.parent.is_inquiry):
```

Comment:
This will also ignore different arguments, maybe it will be safer to just say that the first argument is neither a read nor a write? (also in the comment above)

Reply:
Done